src/Network/Session.php

```diff
@@ -100,7 +100,7 @@ public static function create($sessionConfig = [])
             }
         }
 
-        if (!isset($sessionConfig['ini']['session.cookie_secure']) && env('HTTPS')) {
+        if (!isset($sessionConfig['ini']['session.cookie_secure']) && env('HTTPS') && ini_get("session.cookie_secure") != 1) {
```

Comment:
It seems like this depends on how php is configured. Boolean values of "on" and "off" may be returned as 1 and 0 (or an empty string) respectively while true and false may return actual true/false booleans. more strict testing may cause some configurations to fail.

#### http://php.net/manual/en/function.ini-get.php

*Note: When querying boolean values*
A boolean ini value of off will be returned as an empty string or "0" while a boolean ini value of on will be returned as "1". The function can also return the literal string of INI value.